lib/ASRouter.jsm

```diff
@@ -928,7 +928,7 @@ class _ASRouter {
         target.browser.ownerGlobal.OpenBrowserWindow({private: true});
         break;
       case ra.OPEN_URL:
-        target.browser.ownerGlobal.openLinkIn(action.data.args, "tabshifted", {
+        target.browser.ownerGlobal.openLinkIn(action.data.args, "current", {
```

Comment:
This is going to open the onboarding links in the current tab too, which is unwanted behaviour